doc/CONTRIBUTING.md

```diff
@@ -5,23 +5,29 @@ Look at open issues, especially with label:
 - [junior job](https://github.com/dmacvicar/terraform-provider-libvirt/issues?q=is%3Aissue+is%3Aopen+label%3A%22junior+job%22), 
 - [help wanted](https://github.com/dmacvicar/terraform-provider-libvirt/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
 
+### Makefile devel workflow.
+
+Go in your build dir ```cd $GOPATH/src/github.com/dmacvicar/terraform-provider-libvirt```,
+then  use
+
+```console
+make test
+```
+
```

Comment:
Please remove this section as well. It's unnecessary.

Reply:
yop, done